config/amd64/buildroot-config

```diff
@@ -166,40 +194,42 @@ BR2_TOOLCHAIN_BUILDROOT_LIBC="glibc"
 #
 # BR2_KERNEL_HEADERS_4_4 is not set
 # BR2_KERNEL_HEADERS_4_9 is not set
-# BR2_KERNEL_HEADERS_4_14 is not set
+BR2_KERNEL_HEADERS_4_14=y
 # BR2_KERNEL_HEADERS_4_19 is not set
 # BR2_KERNEL_HEADERS_5_4 is not set
-BR2_KERNEL_HEADERS_5_10=y
```

Comment:
Ah, this copied from 4.14.x. Can you change it back to 5.10 headers on this repo.